pkg/daemon/ceph/client/radosnamespace.go

```diff
@@ -0,0 +1,109 @@
+/*
+Copyright 2022 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package client
+
+import (
+	"encoding/json"
+	"fmt"
+	"syscall"
+
+	"github.com/pkg/errors"
+	"github.com/rook/rook/pkg/clusterd"
+	"github.com/rook/rook/pkg/util/exec"
+)
+
+// CreateRadosNamespace create a rados namespace in a pool.
+// poolName is the name of the ceph block pool, the same as the CephBlockPool CR name.
+func CreateRadosNamespace(context *clusterd.Context, clusterInfo *ClusterInfo, poolName, namespaceName string) error {
+	cName := fmt.Sprintf("%s/%s", clusterInfo.Namespace, clusterInfo.name)
```

Comment:
Instead of logging the namespace/name of the cluster CR, the namespace is always sufficient since we only allow a single cluster in the namespace. I believe @BlaineEXE's earlier comments about logging the fully qualified name was referring to logging the CR name that is being processed instead of logging the cluster CR name. But since the rados namespace belongs to a pool, I think we need to log the poolname/radosNamespace, with the k8s namespace as a separate comment. I've updated the example in the next line with the suggestion that would also apply in a number of other logs as well.

Reply:
fixing logging to log the kubernetes namespace name.